App.js

```diff
@@ -12,46 +6,42 @@ import {
   View
 } from 'react-native';
 
-const instructions = Platform.select({
-  ios: 'Press Cmd+R to reload,\n' +
-    'Cmd+D or shake for dev menu',
-  android: 'Double tap R on your keyboard to reload,\n' +
-    'Shake or press menu button for dev menu',
-});
-
-export default class App extends Component<{}> {
-  render() {
-    return (
-      <View style={styles.container}>
-        <Text style={styles.welcome}>
-          Welcome to React Native!
-        </Text>
-        <Text style={styles.instructions}>
-          To get started, edit App.js
-        </Text>
-        <Text style={styles.instructions}>
-          {instructions}
-        </Text>
-      </View>
-    );
+import { Provider } from 'react-redux';
+import { Navigation } from 'react-native-navigation';
+import configureStore from './app/store/configStore.js';
+
+import colors from './app/config/colors';
```

Comment:
Are we separating library imports and file imports or based on functionality?

Reply:
yes...